### ISSUE TITLE
review calibration error messages

### ISSUE THREAD

**Issue by dagar** [PX4/Firmware#8100]
QGC hides calibration messages (containing "[cal]"), but this also includes important errors, like out of memory. We need to review all calibration code and remove the [cal] prefix from anything critical a user actually needs to see.

https://github.com/PX4/Firmware/blob/master/src/modules/commander/mag_calibration.cpp#L578

**Comment by dagar** [comment]
https://github.com/PX4/Firmware/issues/8098

**Comment by dagar** [comment]
TODO: if USB is connected (extra mavlink running) and calibration doesn't have sufficient memory then suggest disconnecting.

**Comment by bryanmonti** [comment]
Disabling the TELEM2 port also helps free up some memory (~2k). I also removed the IIC LED, didn't measure the memory difference though. I noticed more failures while connected over the telem radios, had good success over USB. More investigation is needed there.

Maybe we can integrate a check that checks for where the external mag is located (ie, UAVCAN or IIC) and disable IO that does not contain a mag. This is more of a temporary patch than a solution.
